unicode.c

```diff
@@ -311,7 +318,7 @@ SQLNativeSqlW (SQLHDBC hdbc, SQLWCHAR *in, SQLINTEGER in_len,
   ODBC_CONNECTION * ConnectionHandle = (ODBC_CONNECTION *) hdbc;
   
   OutputDebugString ("SQLNativeSqlW called.\n");
-  ret = wide_char_to_bytes (in, in_len, &sql_state,  &sql_state_len, NULL);
+  ret = wide_char_to_bytes (in, in_len, &sql_state,  &sql_state_len, ConnectionHandle->charset);
```

Comment:
`ret = wide_char_to_bytes (in, in_len, &sql_state,  &sql_state_len, conn->charset);`